Source/WebInspectorUI/UserInterface/Views/FontVariationDetailsSectionRow.js

```diff
@@ -142,6 +165,87 @@ WI.FontVariationDetailsSectionRow = class FontVariationDetailsSectionRow extends
 
         return 1;
     }
+
+    _showValidity()
+    {
+        if (!this._hasValidValue)
+            this.warningMessage = WI.UIString("Axis value outside of supported range: %s – %s", "A warning that is shown in the Font Details Sidebar when the value for a variation axis is outside of the supported range of values").format(this._formatAxisValueAsString(this._minimumValue), this._formatAxisValueAsString(this._maximumValue));
```

Comment:
i dont really like the look of how this pushes everything to the left, breaking the gridlike consistency of the section (i.e. the inputs no longer horizontally align).  can we move the ⚠️ icon so that it doesnt shift the inputs?

maybe just add a `top: 0;` and remove the `background-color` and `clip-path` so that it shows up in the topright corner instead?

Reply:
That can work when the axis has a human-readable label like in the video demo for this patch because the label creates the negative space for the warning icon to show up (provided the label isn't very long).

But there are variable fonts with unlabeled axes. They just have a tag. The whole label row goes away and the warning icon ends up overlapping the number input again.

We could use a smaller icon that fits in the space between the input and the edge of the panel. 
Making the icon smaller will also reduce its visibility. In that case we may want to keep the yellow background.

Reply:
Here's an attempt at making the icon small enough to fit in the existing gap.
Shown both with and without the yellow background. A warning in the Styles panel for comparison.

@dcrousso @patrickangle what do you think?

## With axis labels
<img width="571" alt="Screenshot 2023-03-02 at 6 14 57 PM" src="https://user-images.githubusercontent.com/105809798/222504678-b026194d-4b5f-46db-a993-177969f47311.png">
<img width="571" alt="Screenshot 2023-03-02 at 6 15 21 PM" src="https://user-images.githubusercontent.com/105809798/222504685-41c07df1-500b-4e81-8bd4-79eab90f4005.png">

## Without axis labels
<img width="571" alt="Screenshot 2023-03-02 at 6 21 07 PM" src="https://user-images.githubusercontent.com/105809798/222504692-2d746362-6f5d-44a8-9c5c-a17d083514ce.png">

<img width="571" alt="Screenshot 2023-03-02 at 6 20 47 PM" src="https://user-images.githubusercontent.com/105809798/222504689-10a79b6a-5e4e-45b2-8962-c45e126e2fb5.png">

Reply:
ooooo i like the look of having the entire row be yellow 🤩

Reply:
I prefer https://user-images.githubusercontent.com/105809798/222506170-93209fe1-1aa1-4490-9a5a-93ddd575c464.png as well with the yellow background remaining and just dropping the explicit shape around the ⚠️ icon.

Reply:
I dropped the clipped background for the warning icon. Now it fits nicely in the gap. 

Additionally, I removed some of the "magic" negative offsets in favor of using `align-items: center` on the grid row.
The tag label, range input, number input and warning icon now align perfectly 👌

I had to use a hack to extract the min/max labels from the grid flow, otherwise they'd contribute to the slider container's height and mess up the baseline. I intend to fix this this by tweaking the grid definition, but ran out of time today. Will follow-up.

Video of the end result:

https://user-images.githubusercontent.com/105809798/222566993-7dc4ce87-32ad-4fc1-b27a-c5319ec56ac3.mov

Reply:
>  I intend to fix this this by tweaking the grid definition, but ran out of time today. Will follow-up.

I'll do this in a follow-up: https://bugs.webkit.org/show_bug.cgi?id=253334